src/fundus/publishers/base_objects.py

```diff
@@ -19,8 +20,17 @@ class PublisherSpec:
     request_header: Dict[str, str] = field(default_factory=dict)
 
 
+class PublisherEnumMeta(EnumMeta):
+    def __str__(self) -> str:
+        representation = f"Publisher Collection {self.__name__} containing {len(self)} publisher(s), including:"
+        publisher: str
+        for publisher in islice(self.__members__.values(), 0, 5):
+            representation += f"\n\t {publisher}"
```

Comment:
I think printing every publisher instead of the first 5 would be appropriate here. I would have expected a sliced list of publishers when printing the entire collection instead.

Reply:
Yeah, I think that makes more sense. I'll switch that around